CONTRIBUTING.md

```diff
@@ -91,7 +91,3 @@ You can run the tests on all repository file with the command:
 ```console
 $ poetry run pre-commit run --all-files
 ```
-
-## License
```

Comment:
Why should we remove this section? Is it a good thing to explicit state this?

Reply:
There is already a section about this: https://github.com/leikoilja/ha-google-home/blob/master/CONTRIBUTING.md#any-contributions-you-make-will-be-under-the-mit-software-license

Reply:
Didn't see that 😊